test/webrtc_test.rb

```diff
@@ -0,0 +1,202 @@
+# frozen_string_literal: true
+
+using_task_library "comms_webrtc"
+import_types_from "iodrivers_base"
+import_types_from "base"
+
+describe OroGen.comms_webrtc.Task do
+    run_live
+
+    attr_reader :task1, :task2
+
+    run_pid = nil
```

Comment:
Even if it does work, this is bad practice. A describe() block essentially defines a class, use instance variables (@rustysignal_pid)